environment-setup/README.md

```diff
@@ -144,11 +143,11 @@ The following example outlines what you would need to do to add an organization
         include:
 
           # Existing
-          - name: "mesur.io-Transmute"
```

Comment:
Can you use any of the other existing ones rather than example? 
Something like mavennet or gs1